pkgs/development/ocaml-modules/magic-mime/default.nix

```diff
@@ -1,27 +1,23 @@
-{ stdenv, lib, fetchzip, ocaml, findlib, ocamlbuild }:
+{ lib, buildDunePackage, fetchFromGitHub }:
 
-let version = "1.0.0"; in
+buildDunePackage rec {
+  pname = "magic-mime";
+  version = "1.1.3";
 
-stdenv.mkDerivation {
-  pname = "ocaml-magic-mime";
-  inherit version;
-
-  src = fetchzip {
-    url = "https://github.com/mirage/ocaml-magic-mime/archive/v${version}.tar.gz";
-    sha256 = "058d83hmxd5mjccxdm3ydchmhk2lca5jdg82jg0klsigmf4ida6v";
+  src = fetchFromGitHub {
```

Comment:
Please use `fetchurl` with that URL: https://github.com/mirage/ocaml-magic-mime/releases/download/v1.1.3/magic-mime-v1.1.3.tbz

Reply:
Done. I am correcting this in every PR but you were faster than my queue. :-)